backend/muster-roll-service/Muster-Roll-Service-1.0.0.yaml

```diff
@@ -208,6 +208,61 @@ components:
           type: object
           description: This can be used to store skillset that is a requirement for Mukta. This is an updatable field from the user's input.
 
+    AttendanceLog:
```

Comment:
Are you using this object anywhere? I don't see any reference to it in the file. 
Were you going to include this in the AttendanceEntry object? 
If you are not referring to this object in the API contract, then it should not be present here.

Reply:
It is used to fetch the attendenceLogs from Attendance service. AttendanceLogResponse attendanceLogResponse = restTemplate.postForObject(uriBuilder.toUriString(),requestInfoWrapper,AttendanceLogResponse.class);